sql_user_check_redshift/database_stats.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+port="5439"
+mode="stats"
+
+while getopts h:u:d:m:o:P:p flag
+do
+    case "${flag}" in
+        h) dbhost=${OPTARG};;
+        u) username=${OPTARG};;
+        d) database=${OPTARG};;
+        m) mode=${OPTARG};;
+        o) outputfile=${OPTARG};;
+        P) password=${OPTARG};;
+        p) port=${OPTARG};;
+    esac
+done
+
+export PGPASSWORD=$password
```

Comment:
Unset this variable whenever the script exits.

Reply:
done, added a cleanup function that will be called before script exists